containers/reverse-proxy/envoy-start

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+cat > /etc/envoy/envoy.yaml << EOF
```

Comment:
Is using `envsubst` an option, then the config below can be in a separate file?
https://stackoverflow.com/questions/14155596/how-to-substitute-shell-variables-in-complex-text-files

Else we can use `sed` to selectively replace environment variables from a file.
e.g. sed -i -e "s/SERVICE_PORT/${SERVICE_PORT}/g" config.yaml

Reply:
Done